schemas/Account.json

```diff
@@ -39,6 +39,10 @@
     "ledger": {
       "description": "A link the the account's ledger",
       "type": "Iri.json"
+    },
+    "fingerprint": {
+      "description": "A fingerprint of the account's client certificate",
+      "type": "string"
```

Comment:
I can add the pattern here for SHA-512, but wouldn't that be `^[0-9A-F]{2}(:[0-9A-F]{2}){,127}$.` -- 512 bits, for hex that's 128 characters